### PR TITLE
Removed Museum, Amongst Other Things.

### PR DESCRIPTION
**Pull request by HurricanePootis**
Read my commits for everything I did, lol. I ain't repeating nothing >:]

Also, there are lots of file changes due to the fact that I received a new set of resources from Ryuke.

### CODE REVIEW

File: thingstodo.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Add back rickmay statue
```
Comment (ZeronTheXeon) [inline thingstodo.txt:1]: Is this file supposed to be pushed up? If so, can this TODO be put in the main README instead?

**Review by ZeronTheXeon — CHANGES_REQUESTED** [review]
I'm on mobile, might be missing it, but have new BSP's been created?

File: thingstodo.txt (hunk 1 above)
Comment (HurricanePootis) [reply thingstodo.txt:1]: yes

**Review by ZeronTheXeon — APPROVED** [review]
Thank you! Looks sweet, I'll squash and merge and leave you to cut a release :)

**Pull request merged**